src/dotnet-scaffolding/Microsoft.DotNet.Scaffolding.Roslyn/Services/MsBuildInitializer.cs

```diff
@@ -45,21 +47,27 @@ private void RegisterMsbuild()
 
             if (!sdkPaths.Any())
             {
-                _logger.LogInformation($"Could not find a .NET SDK at the default locations.");
+                _logger.LogDebug($"Could not find a .NET SDK at the default locations.");
```

Comment:
Should we make this message and the one above it different so we know which check failed based on the message?

Reply:
yes will do, makes sense.